src/Providers/EventServiceProvider.php

```diff
@@ -15,4 +17,8 @@ class EventServiceProvider extends ServiceProvider
             SendEmailVerificationNotification::class,
         ],
     ];
+    public function boot(): void
```

Comment:
```suggestion

    public function boot(): void
```

Please take care about vertical density.